backend/lambdas/process_post/lambda.py

```diff
@@ -0,0 +1,193 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+import os
+import traceback
+
+import demoji
+
+import boto3
+import langchain_core
+from langchain_aws import ChatBedrock
+
+from output_models.models import ExtractedInformation, TopicMatch
+from prompt_selector import get_information_extraction_prompt_selector, get_topic_match_prompt_selector
+
+from aws_lambda_powertools.utilities.typing import LambdaContext
+from aws_lambda_powertools import Logger
+
+MODEL_ID = os.environ['MODEL_ID']
+AWS_REGION = os.environ['AWS_REGION']
+LANGUAGE_CODE = os.environ['LANGUAGE_CODE']
+META_TOPICS_STR = os.environ['LABELS']
+META_SENTIMENTS_STR = os.environ['SENTIMENT_LABELS']
+
+INFORMATION_EXTRACTION_PROMPT_SELECTOR = get_information_extraction_prompt_selector(LANGUAGE_CODE)
+TOPIC_MATCH_PROMPT_SELECTOR = get_topic_match_prompt_selector(LANGUAGE_CODE)
+
+TOPIC_MATCH_MODEL_PARAMETERS = {
+    "max_tokens": 500,
+    "temperature": 0.1,
+    "top_k": 20,
+}
+
+INFORMATION_EXTRACTION_MODEL_PARAMETERS = {
+    "max_tokens": 1500,
+    "temperature": 0.1,
+    "top_k": 20,
+}
+
+bedrock_runtime = boto3.client(
+    service_name="bedrock-runtime",
+    region_name=AWS_REGION
+)
+
+logger = Logger()
+
+langchain_core.globals.set_debug(True)
+
+def unknown_to_empty_text_values(extracted_info: ExtractedInformation):
```

Comment:
minor suggestion: mutating objects is generally not a good practice, it could be better to create a new instance of `ExtractedInformation` and return this new instance.

Reply:
Fixed in [e91f7a0](https://github.com/aws-samples/ai-powered-text-insights/pull/3/commits/e91f7a05ccb41059a49562d086e6824bc3af9ecc)